tasks/create_vms.yml

```diff
@@ -9,6 +9,7 @@
     template: "{{ current_vm.template | default(current_vm.profile.template) | default(omit) }}"
     template_version: "{{ current_vm.template_version | default(current_vm.profile.template_version) | default(omit) }}"
     ballooning_enabled: "{{ current_vm.ballooning_enabled | default(current_vm.profile.ballooning_enabled) | default(omit) }}"
+    host: "{{ current_vm.host| default(current_vm.profile.host) | default(omit) }}"
```

Comment:
Please add space `current_vm.host | `.

Reply:
Fixed.

Reply:
Thanks for the review.

Reply:
Oh just realized if you could also update README